common/defs-flags.go

```diff
@@ -135,6 +135,7 @@ const (
 	// Task Queue flags
 	FlagTaskQueueName           = "Name of the Task Queue."
 	FlagTaskQueueTypeDefinition = "Task Queue type [workflow|activity]"
+	FlagPartitionsDefinition    = "Query for all partitions up to this number"
```

Comment:
Since this may be removed in the future when implemented server-side, can we mark this flag as experimental and add a note?

Reply:
definitely